doc/release-notes/2024-10-03.rst

```diff
@@ -0,0 +1,53 @@
+.. _release_notes_2024_10_03:
+
+Highlights from 2024-10-03 release
+==================================
+
+Enhancements
+------------
+
+- Dist git client moved from Copr to DistGit `PR#3299`_.
+- Jakub started to adding support for Pulp in Copr. We are slowly adding more
+  features to it. Tracking epic issue is `Issue#2533`_.
+- Copr has a nice view of webhook history for each project. You can see what
+  webhooks were sent and from where `PR#3342`_ and `PR#3407`_.
```

Comment:
Maybe mention where users can find the information.

Reply:
Also, I'd like to explicitly mention that the feature was contributed from the community as a response to https://frostyx.cz/posts/do-you-want-to-contribute-to-copr